src/server/index.jsx

```diff
@@ -135,18 +136,20 @@ server
         // Set derivedPageType based on matched route
         derivedPageType = pageType || derivedPageType;
 
+        const toggles = await getToggles(service);
```

Comment:
I'm seeing a lot of  `CONFIG_ERROR`s output to the console when running the tests for these files:

```
2020-08-04 09:46:20.520 error [getToggles/index.js] {
        "event": "config_error",
        "message": {
          "error": "FetchError: invalid json response body at  reason: Unexpected end of JSON input",
          "url": "https://config.test.api.bbci.co.uk/?application=simorgh&service=igbo&__amp_source_origin=http://localhost:7080",
          "service": "igbo"
        }
      }
```

The tests pass as we handle this gracefully. Probably worth checkout out what's going on in here and updating the tests (and adding one to ensure we are getting toggles here)

Reply:
Good spot, I've mocked out getToggles with the defaultToggles return, as we do in other places. It was trying to do the request and failing.